tests/mozdef_util/test_elasticsearch_client.py

```diff
@@ -412,8 +412,8 @@ def test_get_indices(self):
         open_indices = self.es_client.get_open_indices()
         open_indices.sort()
         expected_indices = [self.alert_index_name, self.previous_event_index_name, self.event_index_name, 'test_index']
-        assert all_indices == expected_indices
-        assert open_indices == expected_indices
+        assert all([expected in all_indices for expected in expected_indices])
```

Comment:
I don't think we need to make this change, do you have an example of this test failing? In a clean test environment, we shouldn't have any additional indices being created that we don't know about.

Reply:
I've pointed you to an example of this that I discovered on Slack last week.